tests/test_statement.py

```diff
@@ -41,10 +45,26 @@ def test_first_match_returns(self, graphkb_conn):
         category = statement.categorize_relevance(graphkb_conn, '2')
         assert category == 'therapeutic'
 
-    def test_second_categoary(self, graphkb_conn):
+    def test_second_category(self, graphkb_conn):
         category = statement.categorize_relevance(graphkb_conn, '3')
         assert category == 'diagnostic'
 
+    def test_third_category(self, graphkb_conn):
+        category = statement.categorize_relevance(graphkb_conn, '4')
+        assert category == 'prognostic'
+
+    def test_fourth_category(self, graphkb_conn):
+        category = statement.categorize_relevance(graphkb_conn, '5')
+        assert category == 'pharmacogenomic'
+
+    def test_fifth_category(self, graphkb_conn):
+        category = statement.categorize_relevance(graphkb_conn, '6')
+        assert category == 'pharmacogenomic'
+
+    def test_predisposition_category(self, graphkb_conn):
+        category = statement.categorize_relevance(graphkb_conn, '8')
+        assert category == 'cancer predisposition'
+
     def test_no_match(self, graphkb_conn):
         category = statement.categorize_relevance(graphkb_conn, 'x')
         assert category == ''
```

Comment:
Correct me if I'm understanding this wrong, but the IPR API doesn't support having a blank category (empty string). If this category value is being used for upload you may want to use the category 'unknown' if there is no match.

Reply:
That is handled in the ipr python library, the graphkb library is used by applications outside of ipr  so it can technically have any string as the result